js/jqueryFunctions.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
+  $('#createtask').click(function() {
+    $('form')[0].reset();
+  });
+});
```

Comment:
User input clears without jQuery if you use a `button` instead. A button inside of a form by default acts like a submit input. So when you hit enter, it triggers the function.